webhook_details_test.go

```diff
@@ -0,0 +1,27 @@
+package fivetran_test
+
+import (
+	"context"
+	"testing"
+)
+
+func TestWebhookDetailsE2E(t *testing.T) {
+	webhookId := CreateWebhookAccount(t)
+
+	result, err := Client.NewWebhookDetails().WebhookId(webhookId).Do(context.Background())
+	if err != nil {
+		t.Logf("%+v\n", result)
+		t.Error(err)
+	}
+
+	AssertEqual(t, result.Code, "Success")
+	AssertNotEmpty(t, result.Data.Id)
+    AssertNotEmpty(t, result.Data.Events)
+    AssertNotEmpty(t, result.Data.CreatedAt)
+    AssertNotEmpty(t, result.Data.CreatedBy)
+    AssertNotEmpty(t, result.Data.Type)
+    AssertNotEmpty(t, result.Data.Secret)
+    AssertNotEmpty(t, result.Data.Url)
```

Comment:
We have 9 fields in response object, but checking only 7?
For managed field we could check exact value, not just existence in response.